gooddata-sdk/gooddata_sdk/insight.py

```diff
@@ -72,6 +73,27 @@
 }
 
 
+class BucketType(Enum):
+    UNDEFINED = 0
+    MEASURES = 1
+    ROWS = 2
+    COLS = 3
+
+
+_LOCAL_ID_TO_BUCKET_TYPE = {
+    "measures": BucketType.MEASURES,
+    "attribute": BucketType.ROWS,
+    "columns": BucketType.COLS,
+}
```

Comment:
I am just wondering if it would be suitable to use the default dictionary here to cover BucketType.UNDEFINED.